apps/docs/package.json

```diff
@@ -26,5 +27,11 @@
     "vitepress": "^1.4.0",
     "vue": "catalog:",
     "vue-chartjs": "^5.3.1"
+  },
+  "imports": {
+    "#src/.vitepress/browser-loader.data": {
+      "storybook": "./src/.vitepress/browser-loader.data.mock.ts",
+      "default": "./src/.vitepress/browser-loader.data.ts"
+    }
```

Comment:
Why is this needed?

Reply:
because the file normally runs in Node.js environment and this would throw an error in Storybook context. 
This is the Storybook recommended way to mock the import.

https://vitepress.dev/guide/data-loading